pandas/io/json/_normalize.py

```diff
@@ -328,8 +328,7 @@ def _recursive_extract(data, path, seen_meta, level=0):
 
         if k in result:
             raise ValueError(
-                "Conflicting metadata name {name}, "
-                "need distinguishing prefix ".format(name=k)
+                f"Conflicting metadata name {k}, " "need distinguishing prefix "
```

Comment:
@jyscao an extra pair of parentheses got in here and needs to be removed.  this sometimes happens when running `black`

Reply:
By parentheses, I'm guessing you mean extra pair of `"`? If that's the case, they have been removed.